### PR TITLE
Implement virtual barcode

### PR DESCRIPTION
**Pull request by ConcernedHobbit**
Introduces some changes to eslintrc to prevent conflicts with prettier.

### CODE REVIEW

File: .eslintrc.js
Comment (ConcernedHobbit) [inline .eslintrc.js:1]: Added prettier-config, removed clashing rules

File: package.json [hunk 1]
```diff
@@ -30,13 +30,16 @@
...
+    "canvas": "^2.11.2",
     "class-variance-authority": "^0.6.1",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "date-fns": "^2.28.0",
     "dotenv": "^16.0.0",
     "ejs": "^3.1.7",
+    "eslint-config-prettier": "^9.0.0",
     "express": "^4.17.2",
+    "finnish-bank-utils": "git+https://github.com/vkomulai/finnish-bank-utils.git",
```
Comment (ConcernedHobbit) [inline package.json:42]: Has to be GitHub as latest version is not published to npm

File: common/types.ts
Comment (ConcernedHobbit) [inline common/types.ts:1]: Only actual difference in addition to prettier changes is a // @ts-ignore to allow compilation

File: package.json (hunk 1 above)
Comment (cxcorp) [reply package.json:42]: Can this be pinned to a specific tag or commit?

**Comment by cxcorp** [comment]
Would it be plausible to separate the formatting changes from the code changes?

File: package.json (hunk 1 above)
Comment (ConcernedHobbit) [reply package.json:42]: It can be pinned to a specific commit.

- [ ] Pin to the most recent commit

**Comment by ConcernedHobbit** [comment]
> Would it be plausible to separate the formatting changes from the code changes into a separate commit?

Would be a lot of work for little gain, sorry

**Comment by ConcernedHobbit** [comment]
Hard reset to @cxcorp's branch, thanks!

**Pull request merged**